### ISSUE TITLE
pset.export() crash in python3.8

### ISSUE THREAD

**Issue by christensen5** [OceanParcels/parcels#996]
I have just installed the current master branch of Parcels in a python3.8 venv on a new machine and am seeing the following error when running the NEMO curvilinear co-ordinates example:

```python
Exception ignored in: <function ParticleFile.__del__ at 0x7f0393249ee0>

Traceback (most recent call last):
  File "/home/alexander/venvs/parcelsdrake/lib/python3.8/site-packages/parcels-2.2.3.dev10+g266410ca-py3.8.egg/parcels/particlefile.py", line 189, in __del__

  File "/home/alexander/venvs/parcelsdrake/lib/python3.8/site-packages/parcels-2.2.3.dev10+g266410ca-py3.8.egg/parcels/particlefile.py", line 194, in close

  File "/home/alexander/venvs/parcelsdrake/lib/python3.8/site-packages/parcels-2.2.3.dev10+g266410ca-py3.8.egg/parcels/particlefile.py", line 305, in export

  File "/home/alexander/venvs/parcelsdrake/lib/python3.8/site-packages/numpy/lib/npyio.py", line 416, in load
NameError: name 'open' is not defined
```

If I have correctly read the stack trace, the error occurs after the pset.execute() loop, perhaps during the conversion of the .npy temp files to a netcdf output? If this is not a known issue I can try installing on a venv with an earlier Python version to see if the likely culprit is some change within python3.8 itself.

**Comment by erikvansebille** [comment]
Hi @christensen5 

We are aware of this annoying bug. However, it's not a fatal bug, fortunately. It often happens if the `ParticleFile` object is closed twice, for example if you first close it explicitly `pfile.close()` and then again when the python script is ended

The best way to avoid the bug is to create the `ParticleFile` object inside the `pset.execute()`, so that it is closed at the end of the execute. An alternative is to issue `pfile.export()` after you're done with all executions; that should create the netcdf output file and you will have it regardless of this error message above. In that case, you can therefore ignore it

So in summary: the error message above is a low-impact bug that turns out to be very difficult to fix. If someone has an idea though how to 'properly' fix it; let us know!
